platform/cosmos/api.go

```diff
@@ -41,86 +41,95 @@ func (p *Platform) CurrentBlockNumber() (int64, error) {
 }
 
 func (p *Platform) GetTxsByAddress(address string) (blockatlas.TxPage, error) {
-	srcTxes := make([]Tx, 0)
-
-	tagsList := []string{"recipient", "sender", "delegator", "destination-validator"}
-
-	for _, tag := range tagsList {
-		responseTxes, _ := p.client.GetAddrTxes(address, tag)
-		srcTxes = append(srcTxes, responseTxes...)
-	}
-
-	normalisedTxs := make([]blockatlas.Tx, 0)
-
-	for _, srcTx := range srcTxes {
-		normalisedInputTx, ok := Normalize(&srcTx)
-		if ok {
-			normalisedTxs = append(normalisedTxs, normalisedInputTx)
-		}
+	srcTxs := make([]Tx, 0)
+	tagsList := []string{"transfer.recipient", "message.sender"}
```

Comment:
does this cover delegation transactions too?

Reply:
yes!! the delegations tx return in both tag transfers